ros_ethercat_model/include/ros_ethercat_model/actuator_command_interface.h

```diff
@@ -0,0 +1,67 @@
+/*
+* Copyright (C) 2019 Shadow Robot Company Ltd - All Rights Reserved. Proprietary and Confidential.
+* Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+*/
```

Comment:
This is a public repository and we have licensed it as BSD, so please copy the license header from another file e.g.
https://github.com/shadow-robot/ros_ethercat/blob/kinetic-devel/ros_ethercat_model/include/ros_ethercat_model/ros_ethercat.hpp#L7-L38

Reply:
No need to add author or creation date